content/code_samples/escrow/websocket/account_objects-request.json

```diff
@@ -1,7 +1,7 @@
 {
```

Comment:
`account_objects-request.json` replaces `account_objects_request_sender.json` and serves as the request code sample for both sender and receiver.